src/content/posts/how-to-invest-using-angor/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How to Invest Using Angor
+published: 2024-06-17
+description: 'Angor allows you to browse various projects and invest securely using cryptocurrency. Create a wallet, select a project, submit your investment for approval, and manage your funds with ease.'
+image: './invest.svg'
+tags:  [Bitcoin, Technology, Education]
+category: 'Angor'
+draft: false 
+---
+### Invest in Projects Securely with Angor: A Step-by-Step Guide
+
+Angor is a decentralized crowdfunding platform built on Bitcoin and uses Nostr for enhanced security and transparency. It allows investors to maintain control over their funds and supports direct communication between investors and founders. Here’s a step-by-step guide on how to invest using Angor.
+
+![How to invest using Angor](./assets/how-to-invest.gif)
+
+### How to invest using Angor: A Step-by-step guide to follow along:
+
+### Browse for Projects
+
+To begin investing, you first need to find a project that interests you.
+
+- **Visit the Angor Platform**: Start by going to the Angor website. If you don’t have an account yet, sign up; otherwise, log in to your existing account.
```

Comment:
Agnor doesn't have accounts it is more like wallets.

`If you don’t have a wallet yet, create a wallet to use Angor; otherwise, recover your wallet.`